scripts/amazon-ssm-agent.gpg

```diff
@@ -0,0 +1,30 @@
+-----BEGIN PGP PUBLIC KEY BLOCK-----
```

Comment:
where does this come from? does the SSM team not provide this in any public location?

Reply:
According to the info here https://docs.aws.amazon.com/systems-manager/latest/userguide/verify-agent-signature.html we have to copy it..